test/cmocka/src/math/arithmetic/exponential.c

```diff
@@ -16,6 +16,7 @@
 #include <time.h>
 #include <stdlib.h>
 
+#include <sof/math/exp_fcn_hifi4.h>
```

Comment:
Can include of this hifi4 header be avoided in applications? Just `#include <sof/math/exp_fcn.h>` should be enough. This way there would be need to #include for every xtensa hifi version?

Reply:
Yes, I can remove the HiFi4 header file and include every version of xtensa HiFi.